backend/src/main/java/pt/ulisboa/tecnico/socialsoftware/tutor/question/TopicService.java

```diff
@@ -46,6 +55,23 @@
         return topicRepository.findTopics(course.getId()).stream().sorted(Comparator.comparing(Topic::getName)).map(TopicDto::new).collect(Collectors.toList());
     }
 
+    @Retryable(
+            value = { SQLException.class },
+            backoff = @Backoff(delay = 5000))
+    @Transactional(isolation = Isolation.READ_COMMITTED)
+    public List<TopicDto> findTournamentTopics(int courseExecutionId) {
```

Comment:
This service does not use Tourmanent? It actual only returbs tge course execution available topics, it seems that this method is not necessaty

Reply:
Done